### ISSUE TITLE
Missing Sample Names in lmerge file

### ISSUE THREAD

**Issue by egoetz** [hall-lab/svtools#263]
Hello. I was able to successfully generate a merged vcf for a large number of accessions. However, of over 27,000 SV calls there were 565 calls that did not have SNAME included on their line. These calls were of varying types. I could not find any documentation on why this would be. Could anyone shed some light on this? I need to be able to see where these calls are coming as my set is too large to sort through manually.

**Comment by ernfrid** [comment]
I don't believe we've encountered this before. It seems likely we've hit some sort of parsing error if these calls are intermixed with the header. Which header lines are found after the variant lines? Which are before?

For TS-86, could you clarify what you mean when you say it is listed in the lmerge file's header? Do you mean it is listed in a line starting with `##SAMPLE=<ID=` but isn't listed in any calls? 

Have you checked to see if your 128 calls that are missing the SNAME look to be from this sample?

**Comment by egoetz** [comment]
My research just ended, so I do not have access to the file on me. But if you are interested in this issue from a developer standpoint I would be happy to retrieve it for you. 

TS-86 was on the same line as chrom, pos, type, etc. Just like the accession would be in a regular (unmerged) VCF. 

Yes. The only time that accession appears is on the line that reads "CHROM\tPOS..." It is not found under SNAME at all.

Apologies for the late response. We had to buckle down for a bit there.
